TreeView/__Internal/OrderedListByGetChildrenAsync.razor

```diff
@@ -36,9 +36,15 @@
 
     List<ChildItem> childItems = new List<ChildItem>();
 
+    public async Task Refresh()
+    {
+        await OnInitializedAsync();

```

Comment:
It feels wrong to call lifecycle methods. If we have to do this, can't we reset the `childItems` component here, refactor the method body of `OnInitializedAsync` into a method (e.g. `InitializeChildItems`) and call that method from here and `OnInitializedAsync`?

Reply:
It was my biggest reserve as well. It fells better to have a seperate methods yes